### PR TITLE
setup(css-library): setup tailwind for all apps with global config in ui package

### PR DESCRIPTION
**Pull request by sidarth-23**
… ui package

## Summary

Made UI library a hub of any UI related things by all tailwind configs and css file required, an export by the ui library<

Also configured tailwind to the apps with the global config and is working

- [ ✅ ] Tested
- [ ❌ ] Test Case added - Not applicable
- [ ✅ ] Build Successful
- [ ✅ ] Sufficient Code comments added
- [ ❌ ] Attached Screenshots / Videos - No screenshots for this
- [ ✅ ] All Relevant Documents added

## Depends on

No dependencies

### CODE REVIEW

**Comment by yogai53** [comment]
@sidarth-23 Please resolve conflict.

**Comment by sidarth-23** [comment]
> @sidarth-23 Please resolve conflict.

Resolved

File: packages/ui/tailwind.config.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { Config } from "tailwindcss";
+
```
Comment (yogai53) [inline packages/ui/tailwind.config.ts:2]: Create new project for Tailwind.

**Review by yogai53 — CHANGES_REQUESTED** [review]

File: packages/ui/tailwind.config.ts (hunk 1 above)
Comment (sidarth-23) [reply packages/ui/tailwind.config.ts:2]: I'm done with it. But I'm unable to squash the commits due to git merge in between the commits

File: packages/tailwind-config/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "exports": {
+    "./postcss-config": "./postcss.config.js",
+    "./tw-config": "./tailwind.config.ts",
+    "./css": "./global.css"
+  },
+  "devDependencies": {
+    "tailwindcss": "^3.4.1",
+    "postcss": "^8.4.33",
+    "autoprefixer": "^10.4.17",
+    "typescript": "^5.3.3"
```
Comment (yogai53) [inline packages/tailwind-config/package.json]: Why do we need this?

Comment (yogai53) [reply packages/tailwind-config/package.json]: Please check other dependencies and ensure if it is really needed for the project.

**Review by yogai53 — CHANGES_REQUESTED** [review]

File: packages/tailwind-config/package.json (hunk 2 above)
Comment (sidarth-23) [reply packages/tailwind-config/package.json]: I checked with the projects using it and found that it works only if tailwind is installed. But not sure if it's required in the config package. I'll test it out update the status here

Comment (sidarth-23) [reply packages/tailwind-config/package.json]: The intuition was correct and I did not need the node-modules in tailwind-config package. 

Except for the tailwindcss. I needed it to get the type of config file which is there in tailwindcss npm. Is it worth deleting tailwindcss from the config package and lose type safety for the config.ts file

Comment (sidarth-23) [reply packages/tailwind-config/package.json]: So we are adding in only the tailwindcss dependency for the config package

**Review by yogai53 — APPROVED** [review]

**Review by elizabeththomas92 — APPROVED** [review]

**Pull request merged**